chrome/skin/style-osx.css

```diff
@@ -0,0 +1,174 @@
+/*
+ * Use extreme selector specificity to override the default bindings,
+ * defined in chrome://browser/content/browser.css, which use the single
+ * id selector #PopupAutoCompleteRichResult.
+ */
+popupset#mainPopupSet panel#PopupAutoCompleteRichResult {
+  /*
+   * This odd CSS property connects an XBL file with XUL DOM.
+   * See mdn.io/moz-binding for more.
+   */
+  -moz-binding: url("chrome://universalsearch/content/binding.xml#recommendation-popup");
+}
+
+/*
+ * Same specificity trick here, to override the default bindings for the
+ * urlbar, also defined in chrome://browser/content/browser.css, which use
+ * the single id selector #urlbar.
+ */
+textbox#urlbar {
+  -moz-binding: url("chrome://universalsearch/content/binding.xml#recommendation-urlbar");
+}
+
+popupset#mainPopupSet panel#PopupAutoCompleteRichResult richlistbox {
+  -moz-binding: url("chrome://universalsearch/content/binding.xml#autocomplete-richlistbox");
+}
+
+#universal-search-recommendation {
+  /*made these obvious colors to test whether stylesheet was loading*/
```

Comment:
Oh, yes, sorry for ambiguity. I was specifically talking about the test styling.